mergify_engine/engine/__init__.py

```diff
@@ -136,6 +168,7 @@ async def _ensure_summary_on_head_sha(ctxt: context.Context) -> None:
             return
 
     opened = ctxt.has_been_opened()
+    synchronized = ctxt.has_been_synchronized()
```

Comment:
might not be worth computing this value here if it's not used later because the function returns early or because `previous_summary` is `None`.